src/pages/arbeidsgiver/[narmestelederid]/[oppfolgingsdialogId]/seplanen.tsx

```diff
@@ -7,14 +7,19 @@ import { OppfolgingsplanOversikt } from "../../../../components/seplanen/Oppfolg
 import { beskyttetSideUtenProps } from "../../../../auth/beskyttetSide";
 import { SendTilGodkjenningAG } from "../../../../components/seplanen/sendtilgodkjenning/SendTilGodkjenningAG";
 import { useAktivPlanAG } from "../../../../api/queries/arbeidsgiver/oppfolgingsplanerQueriesAG";
+import { GodkjennEgenPlanAG } from "../../../../components/seplanen/sendtilgodkjenning/GodkjennEgenPlanAG";
 
 const Seplanen: NextPage = () => {
   const aktivPlan = useAktivPlanAG();
-
+  const isOwnLeader =
+    aktivPlan?.arbeidsgiver?.naermesteLeder?.fnr ===
```

Comment:
Her vil det bli true hvis aktivPlan er nullish, `undefined === undefined` => `true`

Reply:
Stemmer. Fikser det samme for SM.